src/modules/leaderboard.ts

```diff
@@ -0,0 +1,118 @@
+import { Sequelize, Op } from "sequelize";
+
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+
+async function updateLeaderboardRow(address: string, property: string, value: number) {
+  const userLeaderboard = await db.leaderboard.findOne({
+    where: { address }
+  });
+
+  if(userLeaderboard) {
+    userLeaderboard[property] = value;
+
+    await userLeaderboard.save();
+  } else
+    await db.leaderboard.create({
+      address,
+      [property]: value,
+    });
+}
+
+/**
+ * Update leaderboard bounties quantity. If the parameter is not passed it will count all bounties.
+ */
+async function updateLeaderboardBounties(state?: "canceled" | "closed") {
+  try {
+    const bountiesOfCreators = await db.issues.findAll({
+      group: ["creatorAddress"],
+      attributes: ["creatorAddress", [Sequelize.fn("COUNT", "creatorAddress"), "id"]],
+      raw: true,
+      ... state ? {
+        where: {
+          state
+        }
+      } : {}
+    });
+
+    if (!bountiesOfCreators.length) 
+      return logger.info(`Leaderboard: updateLeaderboardBounties ${state} no bounties found`);
```

Comment:
don't return a logging instance. either return something useful for the function that calls this function or return void instead,

```ts
if (!something) {
  logger.info(`yadda`);
  return;
}
```